### DISCUSSION TITLE
Possible To Use Manifest (PWA) For Kiosk Page?

### DISCUSSION THREAD

**Discussion by gary-hammer** [poblabs/weewx-belchertown#825] [Q&A]
I have an Android tablet that is used solely as a display.
Currently, I use the PWA ability of the Belchertown skin on the tablet.
This works, but the formatting is off somewhat.
The new kiosk page would be great, but displaying that on the tablet needs the browser (Chrome) open which brings address bar, tabs, etc.

If I could get the kiosk page via the manifest that would be ideal.

**Comment by steepleian** [comment]
If your tablet is an iPad there is a free app called Kiosk which has address bars, tabs etc removed.IanSent from my iPhoneOn 13 Oct 2022, at 11:26, gary-hammer ***@***.***> wrote:﻿
I have a tablet that is used solely as a display.
Currently, I use the PWA ability of the Belchertown skin on the tablet.
This works, but the formatting is off somewhat.
The new kiosk page would be great, but displaying that on the tablet needs the browser open which brings address bar, tabs, etc.
If I could get the kiosk page via the manifest that would be ideal.

**Reply by gary-hammer** [reply]
Thanks, but I have an Android tablet.
Updated the question.

**Comment by ThePeePs** [comment]
@gary-hammer in [this video](https://www.youtube.com/watch?v=llYMjhC2w5k) he uses a fire table (runs android) to use as a wall mounted display for home assistant. Maybe that will help you?  I also have found (but haven't tested yet) a [card mod](https://community.home-assistant.io/t/card-mod-add-css-styles-to-any-lovelace-card/120744/2399) to scale an iframe lovelace card.

If you're not doing home assistant, I don't see why the kiosk browser he talks about won't work for the kiosk page.

**Reply by gary-hammer** [reply]
Well, I'd rather find a way to run Chrome full screen on a tablet or modify/use the manifest than setup HA for this. I have no other home automation. I gave using HA a shot for integrating my Davis weather station and my irrigator.
After a few days of not understanding HA and it's setup, I chucked it and let the irrigator do its thing.

**Reply by ThePeePs** [reply]
I was more talking about the browser he was using on the tablet.  it should work for displaying the kiosk page, unless there's a way to put chrome into kiosk mode in Android

**Reply by gary-hammer** [reply]
Ah!
I had to use CC to get the name of it and it does work well in the free edition.
I have a Galaxy Tab A7 Lite 8.7" with 1340 x 800 screen size.
The kiosk browser does the best job in trying to display the Windy inset. Not scrollable though as intended.
The normal Belchertown PWA does not have it in the viewable area, but it is of course there if you scroll.
The kiosk mode in Chrome does the least, and also loses the Sun & Moon info.
See attached photos.
I guess I can live with the Windy/radar not being functional on the tablet but we do like that when storms are heading our way.

![kiosk-browser_cr](https://user-images.githubusercontent.com/12861605/195620305-72d77cc6-e5bb-4693-9eda-88324f522a07.jpg)
![kiosk-chrome_cr](https://user-images.githubusercontent.com/12861605/195620307-8a037846-6e71-4b12-be81-543f4afd2b91.jpg)
![kiosk-PWA_cr](https://user-images.githubusercontent.com/12861605/195620310-0be2aa17-57e3-4889-aa8e-6f802ef04e14.jpg)

**Reply by ThePeePs** [reply]
Maybe try scaling down the page a bit, since it's formatted to be 1200 px wide. but glad you were able to find a solution.

**Reply by gary-hammer** [reply]
It wouldn't seem to be the width, it is 1340 px wide.
But, where shall I change the dimensions and I'll play with it a bit more.

Thanks for the tip on the browser.

**Comment by michaelundwd** [comment]
I don't know if this helps, but I have a Nexus 10 tablet running Android solely used for display.  It's some time since I did it, but I think I modified the header file and style.css to show me what I wanted without having to scroll the screen.  Take a look at lordshipweather.uk and set the screen size to 1280* 800 in developer mode (I use Chrome).  Happy to share my code if that would help you.
Michael

**Reply by gary-hammer** [reply]
Thanks, I'll take a look.

**Comment by michaelundwd** [comment]
Forgot to say I use fully-kiosk as the browser on the tablet, so that only the weather website is displayed, no address bar or tabs etc.

**Reply by gary-hammer** [reply]
Yes, I found that one thanks to an earlier post.
